test/e2e/quick_start_test.go

```diff
@@ -198,7 +198,7 @@ var _ = Describe("When following the Cluster API quick-start check finalizers re
 	})
 })
 
-var _ = Describe("When following the Cluster API quick-start with ClusterClass check finalizers resilience after deletion [ClusterClass]", func() {
+var _ = Describe("When following the Cluster API quick-start with ClusterClass check finalizers resilience after deletion [ClusterClass] [Aditya]", func() {
```

Comment:
cleanup issue, will remove after the current e2e full job is concluded.

Reply:
https://github.com/kubernetes-sigs/cluster-api/compare/1f087aa3240e24e20c37d35f6cfabae88ed82aa8..ba855c950f6da506d48b8f28dd5ef5e3fe667a17